packages/sitecore-jss-proxy/src/middleware/monitoring/healthcheck-middleware.test.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import request from 'supertest';
+import { healthCheckMiddleware } from './index';
+
+describe('editingRouter - /editing/render', () => {
+  const app = express();
+
+  it('should handle request', async () => {
+    app.get('/api/halthz', healthCheckMiddleware());
```

Comment:
Not required to be updated, but still
```suggestion
    app.get('/api/healthz', healthCheckMiddleware());
```